plugins/inputs/mongodb/mongostat.go

```diff
@@ -1356,6 +1361,26 @@ func NewStatLine(oldMongo, newMongo MongoStatus, key string, all bool, sampleSec
 					returnVal.ReplLag = lag
 				}
 			}
+
+			// Prepartions for the average health state of the replica-set
+			var replMemberCount int = len(newReplStat.Members)
+			var replMemberHealthyCount int = 0
```

Comment:
Please use
```suggestion
			replMemberCount := len(newReplStat.Members)
			replMemberHealthyCount := 0
```

to make the linter happy and make the code look nicer. ;-)

Reply:
changed with e07ef0a1e